cli/azd/pkg/auth/manager.go

```diff
@@ -593,10 +593,6 @@ func (m *Manager) saveSecret(tenantId, clientId string, ps *persistedSecret) err
 type CredentialForCurrentUserOptions struct {
 	// The tenant ID to use when constructing the credential, instead of the default tenant.
 	TenantID string
-
-	// If true, TenantID is only used if the tenant wasn't explicitly specified
-	// at the time of login.
-	PreferFallbackTenant bool
```

Comment:
This is no longer needed after recent changes to detect service principal logons.